pkg/v20/resource/app/desired.go

```diff
@@ -60,19 +60,19 @@ func (r *Resource) newApp(clusterConfig v1alpha1.ClusterGuestConfig, appSpec key
 			Namespace: appSpec.Namespace,
 			Version:   appSpec.Version,
 
-			Config: g8sv1alpha1.AppSpecConfig{
+			Config: &g8sv1alpha1.AppSpecConfig{
```

Comment:
Not sure where this is coming from. Fixing to get going.

Reply:
It's from https://github.com/giantswarm/apiextensions/pull/283. Using the pointer results in cleaner app CRs but it does touch a lot of places.